tests/mock_sftpserver.h

```diff
@@ -35,6 +35,7 @@ DECL_MOCK(sftp_reply_names_add);
 DECL_MOCK(sftp_reply_handle);
 DECL_MOCK(sftp_get_client_message);
 DECL_MOCK(sftp_client_message_free);
+DECL_MOCK(sftp_client_message_get_submessage);
```

Comment:
Looks like this is not needed based on the tests.

If it's needed in the future, it also needs to be added to "c_mock_defines.cmake"